pkg/completion/completion.go

```diff
@@ -0,0 +1,27 @@
+package completion
+
+import (
+	"github.com/shipyard/shipyard-cli/pkg/client"
+
+	"github.com/spf13/cobra"
+)
+
+type Completion struct {
+	client client.Client
+}
+
+func New(cl client.Client) Completion {
+	return Completion{client: cl}
+}
+
+func (c Completion) EnvironmentUUIDs(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
```

Comment:
I don't understand the dogsledded args here.  Can you fill me in on why they're present?

Reply:
This method is a value of `ValidArgsFunction` (a field on`cobra.Command`) whose signature requires these parameters.